sites/hashdev/src/_pages/blog/4_announcing-error-stack.mdx

```diff
@@ -264,21 +262,21 @@ We don't have space here to talk about all of the features of `error-stack`, but
 - Just like [`eyre`](https://crates.io/crates/eyre), the crate is able to set its own `Display` and `Debug` hooks that are called when printing the `Report`, which means you can write your own custom formatting and do other cool things
 - We have in-built support for backtraces on the nightly channel and feature-flagged support for `tracing` to capture a `SpanTrace`
 
-## The Future of `error-stack`
+### The future of `error-stack`
 
 As previously noted, we are still in the process of migrating our own codebase to `error-stack` and we intend to make further improvements as we apply the crate to more of hEngine and our internal code at [HASH](https://hash.ai/).
 
-We will switch to the provider implementation in `core::any` in version 0.2.0 (the main reason why this API currently requires the nightly release channel).
+During development of the library we included a `Provider` implementation while waiting for the implementation of the [`Provider` RFC](https://rust-lang.github.io/rfcs/3192-dyno.html) to be merged. We will be updating the library to swap to the `core::any` implementation in version 0.2.0 (This is the reason why this API currently **requires the nightly release channel**).
 
 We cannot guarantee full backward compatibility until this crate hits `v1.0.0`. However, we will aim to avoid breaking changes, and in the event of any will announce them beforehand and provide a migration guide.
 
 Contributions are greatly welcome. View the [project code on GitHub](https://github.com/hashintel/hash/tree/main/packages/libs/error-stack) and, if you like it, please star the repository!
 
-## Acknowledgments
+### Acknowledgments
```

Comment:
```suggestion
## Acknowledgments
```

These aren't part of "What is error-stack" (I think I accidentally suggested you should change them)